Plone 5: disable resource registries (performance)

jone opened the pull request:
In Plone 5, the resource registries are cooked when the resource registry viewlets are rendered. Cooking the bundles takes a lot of time. Since ``ftw.testbrowser`` does nothing with JavaScript or CSS, cooking of resources is disabled by default for performance improvement. That means that ``<script>`` and ``<styles>`` tags are missing in the HTML. This can make the tests up to 4-5 times faster.

ℹ️ This goes into `1.x`. I'll forward-port it to `master`, but I need it in 1.x.

buchi commented on docs/source/userdoc.rst:
```diff
@@ -648,3 +648,32 @@ and asserted there.
+
+In Plone 5, the resource registries are cooked when the resource registry viewlets are rendered.
+Cooking the bundles takes a lot of time.
+Since ``ftw.testbrowser`` does nothing with JavaScript or CSS, we cooking the resources is disabled
```
... or CSS, ~~we~~ cooking __of__ the resources ....

buchi commented on ftw/testbrowser/core.py:
```diff
@@ -49,6 +49,11 @@
     from plone.app.testing import TEST_USER_NAME
     from plone.app.testing import TEST_USER_PASSWORD
 
+HAS_PLONE_5 = pkg_resources.get_distribution('Plone').version[:2] == '5.'
```
There's a helper for that in `Products.CMFPlone.utils.getFSVersionTuple`

jone commented:
👍 merci, updated.
I have switched the base and merge target from `1.x` to `master` because tests do not reliably work on `1.x`. It's ok to ship it with `2.0`.
